ledger/testing/randomAccounts.go

```diff
@@ -187,7 +187,7 @@ func RandomAppLocalState() basics.AppLocalState {
 }
 
 // RandomFullAccountData generates a random AccountData
-func RandomFullAccountData(rewardsLevel uint64, knownCreatables map[basics.CreatableIndex]basics.CreatableType, lastCreatableID uint64) (basics.AccountData, map[basics.CreatableIndex]basics.CreatableType, uint64) {
+func RandomFullAccountData(rewardsLevel uint64, lastCreatableID *basics.CreatableIndex, assets map[basics.AssetIndex]struct{}, apps map[basics.AppIndex]struct{}) basics.AccountData {
```

Comment:
Did you make leastCreatableID a pointer so that you could find out the app ids that were created?  I'd rather see it left as a value, but add the final value as an extra return. (or you can guess the ids in the caller based on the number of assets and apps?)

Reply:
wait... this is also not wut I did... I have to apologize for the introducing diffs from master now...

Reply:
ok, probably somebody else needed to do what I described.  ignore it if it's not your code. I thought maybe you needed to know ap ids for some test where you get the boxes for a specific appid.